go/test/endtoend/tabletmanager/replication_manager/tablet_test.go

```diff
@@ -164,11 +164,12 @@ func waitForSourcePort(ctx context.Context, t *testing.T, tablet cluster.Vttable
 	return fmt.Errorf("time out before source port became %v for %v", expectedPort, tablet.Alias)
 }
 
-func getSidecarDbDDLQueryCount(tablet *cluster.VttabletProcess) (int64, error) {
+func getSidecarDBDDLQueryCount(tablet *cluster.VttabletProcess) (int64, error) {
 	vars := tablet.GetVars()
-	val, ok := vars["SidecarDbDDLQueryCount"]
+	key := "SidecarDBDDLQueryCount"
```

Comment:
Should this key be a constant? I see we are using it in several places.

Reply:
Fixed